### PR TITLE
Use copy.copy on the generated items.

### PR DESCRIPTION
**Pull request by nipunn1313**
This ensures that each one hashes differently, which allows it to
work with funcmanage - which rolls the array into a dict and back.

Additionally, add py3.7 to the test matrix

### CODE REVIEW

**Comment by nipunn1313** [comment]
CC @euresti ! Noticed that flakefinder wasn't working when funcmanage was enabled. This fixes it.

File: pytest_flakefinder.py [hunk 1]
```diff
@@ -72,7 +73,7 @@ def pytest_collection_modifyitems(self, items):
         # Also we want to @tryfirst so that we go before randomizing the list.
         for item in list(items):
             if not getattr(item.function, '_pytest_duplicated', None):
-                items.extend([item] * (self.flake_runs - 1))
+                items.extend([copy.copy(item)] * (self.flake_runs - 1))
```
Comment (euresti) [inline pytest_flakefinder.py]: Doesn't that stick the same copy of `item` multiple times.  You might want a 
`copy.deepcopy([item] * (self.flake_runs - 1)` instead?

**Comment by nipunn1313** [comment]
tests are broken because newer versions of pytest don't work well with unittest =\
I just submitted a PR to get the tests working again

File: pytest_flakefinder.py (hunk 1 above)
Comment (nipunn1313) [reply pytest_flakefinder.py]: sadly, you can't deepcopy item because it has some stuff inside that doesn't support it

Comment (nipunn1313) [reply pytest_flakefinder.py]: good catch though - I will fix it up.

**Pull request merged**